README.md

```diff
@@ -10,15 +10,29 @@ are pretty much free to use it however you want in whatever way.
 
 For more info, see <http://logstash.net/>
 
-## logstash-contrib
+## Logstash Plugins
 ### AKA "Where'd that plugin go??"
 
-Since version 1.4.0 of Logstash, some of the community-contributed plugins were
-moved to a new home in the
-[Elasticsearch logstash-contrib repo](https://github.com/elasticsearch/logstash-contrib).
-If you can't find a plugin here which you've previously used, odds are it is now
-located there. The good news is that these plugins are simple to install using the
-[Logstash manual plugin installation script](http://logstash.net/docs/latest/contrib-plugins).
+Since version **1.5.0 beta1 (and current master)** of Logstash, *all* plugins have been separated into their own
+repositories under the [logstash-plugins](https://github.com/logstash-plugins) github organization. Each plugin is now a self-contained Ruby gem which
+gets published to RubyGems.org. Logstash has added plugin infrastructure to easily maintain the lifecyle of the plugin.
+For more details and rationale behind these changes, see our [blogpost](http://www.elasticsearch.org/blog/plugin-ecosystem-changes/). 
+
+[Elasticsearch logstash-contrib repo](https://github.com/elasticsearch/logstash-contrib) will slowly be deprecated. We
```

Comment:
stronger and present-tense language: "is deprecated", not "will slowly be deprecated" - thoughts?